orchestra/tests/unit/specs/mistral/test_workflow_spec_validate.py

```diff
@@ -16,36 +16,15 @@
 class WorkflowSpecValidationTest(base.MistralWorkflowSpecTest):
 
     def test_success(self):
-        wf_def = """
-            version: '2.0'
-
-            sequential:
-                description: A basic sequential workflow.
-                tasks:
-                    task1:
-                        action: std.noop
-                        on-success:
-                            - task2
-                    task2:
-                        action: std.noop
-                        on-success:
-                            - task3
-                    task3:
-                        action: std.noop
-        """
-
+        # load definition from fixture: tests/fixtures/workflows/mistral/xxx.yaml
+        wf_def = self.get_wf_def("sequential-success")
         wf_spec = self.instantiate(wf_def)
 
         self.assertDictEqual(wf_spec.inspect(), {})
 
     def test_missing_task_list(self):
-        wf_def = """
-            version: '2.0'
-
-            sequential:
-                description: A basic sequential workflow.
-        """
-
```

Comment:
We only put WF definition to `orchestra/tests/fixtures/workflows/mistral` if it will be commonly used.  If the WF definition is only used in only 1 test, we inline the definition in the test for easier references. Also, this change doesn't seem to have anything to do w/ this PR.